packages/edit-site/src/components/page-actions/index.js

```diff
@@ -9,18 +9,20 @@ import { moreVertical } from '@wordpress/icons';
  * Internal dependencies
  */
 import TrashPageMenuItem from './trash-page-menu-item';
+import EditPageMenuItem from '../rename-menu-item';
 
-export default function PageActions( { postId, toggleProps, onRemove } ) {
+export default function PageActions( { post, toggleProps, onRemove } ) {
```

Comment:
Is there a reason to change this prop? If possible its always better to pass simpler props as it prevents unnecessary rerenders in the component.